src/accumulator/stump.rs

```diff
@@ -0,0 +1,139 @@
+use super::types;
+
+#[derive(Debug, Clone)]
+pub struct Stump {
+  leafs: u64,
+  roots: Vec<bitcoin_hashes::sha256::Hash>
+}
+
+impl Stump {
+  /// Creates an empty Stump
+  pub fn new() -> Self {
+    Stump {
+      leafs: 0,
+      roots: Vec::new()
+    }
+  }
+  /// Modify is the external API to change the accumulator state. Since order
+  /// matters, you can only modify, providing a list of utxos to be added, 
+  /// and txos (@TODO) to be removed, along with it's proof. Either may be
+  /// empty.
+  ///# Example
+  /// ```
+  ///   use rustreexo::accumulator::stump::Stump;
+  ///   let mut s = Stump::new();
+  ///   let utxos = vec![];
+  ///   let stxos = vec![];
+  ///   s.modify(&utxos, &stxos);
+  /// ```
+  pub fn modify(&mut self, utxos: &Vec<bitcoin_hashes::sha256::Hash>, _stxos: &Vec<bitcoin_hashes::sha256::Hash>) {
```

Comment:
Stump wouldn't be able to take in stxos for deletion because it doesn't know where each of the txos would be located in the tree. Stump should take in a vector of u64s.

Reply:
Sure. Didn't put too much effort on that, since deletion isn't implemented yet, but 027fe2d fixes this by changing modify's signature.